move/axelar_gateway/sources/gateway.move

```diff
@@ -600,6 +602,9 @@ fun test_take_approved_message() {
         destination_id,
         payload,
     );
+
+    utils::single_event<events::MessageExecuted>();
```

Comment:
```suggestion
    assert_event<events::MessageExecuted>();
```
can you rename it to `assert_event` or `assert_emitted_event`? easier to understand